include/ros1_bridge/factory.hpp

```diff
@@ -241,34 +241,37 @@ class ServiceFactory : public ServiceFactoryInterface
   using ROS2Response = typename ROS2_T::Response;
 
   void forward_2_to_1(
-    ros::ServiceClient client, const std::shared_ptr<rmw_request_id_t>,
+    ros::ServiceClient & client, rclcpp::Logger logger, const std::shared_ptr<rmw_request_id_t>,
```

Comment:
The added reference seems to be unrelated to the logging change. Please describe why it was added.

Reply:
Thanks, carried over from the unrelated changes, purging.